src/embed.js

```diff
@@ -20,26 +21,34 @@ var MODES = {
   'vega-lite': 'vega-lite'
 };
 
+var VERSION = {
+  'vega':      vega.version,
+  'vega-lite': vl.version
+}
+
 var PREPROCESSOR = {
   'vega':      function(vgjson) { return vgjson; },
   'vega-lite': function(vljson) { return vl.compile(vljson).spec; }
```

Comment:
How should we handle the warnings from vega-lite?

So far, it shows warning messages when `vljson` is something wrong.
For example, 
```JSON
{
  "$schema": "https://vega.github.io/schema/vega-lite/v2.0.1.json",
  "description": "A simple bar chart with embedded data.",
  "data": {
    "values": [
      {"a": "A","b": 28}, {"a": "B","b": 55}, {"a": "C","b": 43},
    ]
  },
  "mark": "xx",
  "encoding": {
    "x": {"field": "a", "type": "ordinal"},
    "y": {"field": "b", "type": "quantitative"}
  }
};

```
Vega-embed with the above spec returns the below.

![image](https://cloud.githubusercontent.com/assets/6269456/22616812/f674b7a0-ea69-11e6-9f70-3215fb528475.png)